src/main/java/org/bricolages/streaming/preflight/domains/DateDomain.java

```diff
@@ -12,33 +12,19 @@
 import lombok.*;
 
 @JsonTypeName("date")
-@MultilineDescription("Date time")
+@MultilineDescription("Date domain")
```

Comment:
[nits] I think "domain" modifier is not needed by convention. (though just "Date" is not explanatory enough...)